### PR TITLE
dev: unify make install-sentry-dev -> install-dev

### PR DESCRIPTION
**Pull request by joshuarli**
See https://github.com/getsentry/develop/pull/36#discussion_r429463007.

There's no reason for this kind of inconsistency. Also, less stuff to type is nice.

getsentry: https://github.com/getsentry/getsentry/pull/3944

### CODE REVIEW

**Review by billyvg — COMMENTED** [review]
is `dev` too generic since this is specifically pip/python

**Comment by joshuarli** [comment]
> is `dev` too generic since this is specifically pip/python

Good point, how about `install-py-dev`?

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,7 @@ YARN := ./bin/yarn
 
 bootstrap: develop init-config run-dependent-services create-db apply-migrations
 
-develop: ensure-pinned-pip setup-git install-yarn-pkgs install-sentry-dev
+develop: ensure-pinned-pip setup-git install-yarn-pkgs install-dev
```
Comment (markstory) [inline Makefile:7]: Does getsentry also have a develop target?

Comment (joshuarli) [reply Makefile:7]: Yeah, why?

**Review by billyvg — APPROVED** [review]
consistency is good

File: Makefile (hunk 1 above)
Comment (markstory) [reply Makefile]: Just wanted to double check that there was a consistent 'install everything' make recipie

**Pull request merged**